web-registry/src/components/molecules/BatchMetadata.tsx

```diff
@@ -10,34 +10,36 @@ export const BatchMetadata: React.FC<{
   data?: BatchMetadataLD;
   sx?: SxProps<Theme>;
 }> = ({ sx, data }) => {
-  const monitoringReport = data?.['regen:batchMonitoringReport'];
-  const verificationReport = data?.['regen:batchVerificationReport'];
+  const additionalCertifications =
+    data?.['regen:additionalCertifications']?.['@list'];
```

Comment:
I believe we might not actually need the @list, see https://github.com/regen-network/regen-registry-standards/pull/15

Reply:
Ok, I understand how this works now. Fixed.